frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -424,7 +425,40 @@ const allHostTableHeaders: IDataColumn[] = [
       />
     ),
     accessor: "public_ip",
-    Cell: (cellProps: ICellProps) => <TextCell value={cellProps.cell.value} />,
+    Cell: (cellProps: ICellProps) => {
+      if (cellProps.cell.value) {
+        return <TextCell value={cellProps.cell.value} />;
+      }
+      return (
+        <>
+          <span
+            className="text-cell text-muted tooltip"
+            data-tip
+            data-for={"public-ip-tooltip"}
+          >
+            ---
+          </span>
+          <ReactTooltip
+            place="bottom"
+            effect="solid"
+            backgroundColor="#3e4771"
+            id={"public-ip-tooltip"}
```

Comment:
I just did something like this myself and encountered a bug that I think would crop up here – if there are more than 1 of these tooltips on a page (which there may well be on the host table, or any table), they'll have the same id if set up like this, which will cause some weird rendering behavior. `_.uniqueId()` works well for this

Reply:
Comin in clutch @jacobshandling

Reply:
id={`public-ip__${cellProps.row.original.id}`} to match rest of hosttableconfig